spec/models/manageiq/providers/amazon/regions_spec.rb

```diff
@@ -15,4 +15,19 @@
       expect(described_class.names).not_to include('us-gov-west-1')
     end
   end
+
+  context "adds additional regions from settings" do
+    it "includes regions from additional_regions" do
+      custom_region = {
+        "my-custom-region" => {
+          :name        => "My Custom Region",
+          :hostname    => "localhost",
+          :description => "Custom Region"
+        }
+      }
+
+      allow(Settings.ems.ems_amazon).to receive(:additional_regions).and_return(custom_region)
```

Comment:
Switched to `stub_settings`

I had pictured a hash rather than an array which matches the way `REGIONS` is structures so e.g. this was my config addition:
```
  :ems_amazon:
    :disabled_regions: []
    :additional_regions:
      :custom-region-2:
        :description: 2nd Custom Region
        :hostname: localhost
        :name: Second Custom Region
      :custom-region-1:
        :description: 1st Custom Region
        :hostname: localhost
        :name: First Custom Region
```

Would this be better as an array?